.ghci

```diff
@@ -19,3 +19,4 @@
 
 :set -XFlexibleInstances
 :set -XRankNTypes
+:set -XStandaloneDeriving
```

Comment:
This specific line is `ghci` config. It enables the extension when invoking the interpreter (common in my workflow). It was also added to the `default-extensions` field of the cabal file, enabling it any time cabal builds the project -- that's more of the "project-wide pragma", I think.